From edb13e8b5e9d10f14f0941884a748cc86e8b40f3 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Sun, 15 Jan 2017 13:40:41 -0800
Subject: [PATCH] sf: Silence 'Timestamp x seems implausible' logspam

Something is funky with the timestamps used in our camera's EIS routines;
they're using CLOCK_BOOTTIME instead of CLOCK_MONOTONIC, which causes
these surfaceflinger warnings to be spammed to the log. These warnings are
not fatal and there are no a/v sync issues in the camera, so silence them
in order to cut out their unneeded overhead.

Change-Id: I798d34d05139e810f9c0bdc1188e2e5cb3de8ede
---
 services/surfaceflinger/Layer.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/services/surfaceflinger/Layer.cpp b/services/surfaceflinger/Layer.cpp
index 6a9cdb7..c1a3184 100644
--- a/services/surfaceflinger/Layer.cpp
+++ b/services/surfaceflinger/Layer.cpp
@@ -1196,9 +1196,6 @@ bool Layer::shouldPresentNow(const DispSync& dispSync) const {
 
     // Ignore timestamps more than a second in the future
     bool isPlausible = timestamp < (expectedPresent + s2ns(1));
-    ALOGW_IF(!isPlausible, "[%s] Timestamp %" PRId64 " seems implausible "
-            "relative to expectedPresent %" PRId64, mName.string(), timestamp,
-            expectedPresent);
 
     bool isDue = timestamp < expectedPresent;
     return isDue || !isPlausible;
-- 
2.9.3

